services/ui_backend_service/frontend.py

```diff
@@ -30,11 +30,11 @@ def __init__(self, app):
         for filepath in static_files:
             filename = filepath[len(static_ui_path) + 1:]
             app.router.add_route(
-                '*', f'/{filename}', self.serve_file(filename))
```

Comment:
Setting "*" method clashes with `aiohttp_cors`, because it overlaps with the `OPTIONS` method needed for preflight CORS handling.

I'm assuming that static stuff only needs `GET`.  When I tested this, it seems to work fine.  Would love to hear any tribal knowledge about the reasoning behind existing `*` catch all route.